WebRoot/css/zm.css

```diff
@@ -1493,19 +1542,26 @@ INPUT[type="text"].search_input-expanded:focus {
 
 /* Styles to show addresses in bubbles in input fields */
 
-.addrBubbleContainer,
+.addrBubbleContainer {
```

Comment:
`. addrBubbleContainer ` class is very generic in ZWC and is used in many other field. For eg search input widget, in share folder dialog etc. So can you please confirm if this rule, doesn't affects other components look and feel.  If so, you can target more specific rule. 

Thanks